extendr-api/src/wrapper/logicals.rs

```diff
@@ -146,3 +146,23 @@ impl std::fmt::Debug for Logicals {
         }
     }
 }
+
+impl TryFrom<Vec<i32>> for Logicals {
+    type Error = Error;
+
+    fn try_from(value: Vec<i32>) -> std::result::Result<Self, Self::Error> {
+        Ok(Self {
+            robj: <Robj>::try_from(value)?,
+        })
+    }
+}
```

Comment:
This type conversion also needs a test. Does this type conversion return `Error` if an i32 value is not 1, 0 or i32::MIN (I think that's the NA value)? Or would that be converted to `Rbool::na()`?